bundle/render/render_text_output.go

```diff
@@ -94,6 +108,9 @@ func buildTrailer(diags diag.Diagnostics) string {
 	if warnings := len(diags.Filter(diag.Warning)); warnings > 0 {
 		parts = append(parts, color.YellowString(pluralize(warnings, "warning", "warnings")))
 	}
+	if infos := len(diags.Filter(diag.Info)); infos > 0 {
+		parts = append(parts, color.BlueString(pluralize(infos, "info", "infos")))
+	}
```

Comment:
One "info" and two "infos" sounds weird.

If we're emitting a recommendation, we can call it "recommendation". WDYT?

Reply:
Done. Added a new enum for a recommendation to `diag.Diagnostics`